recipes/pestpp/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir -p build && cd build
+
+cmake ${CMAKE_ARGS} \
+      -D CMAKE_BUILD_TYPE=Release \
+      -D BUILD_SHARED_LIBS=OFF \
```

Comment:
```suggestion
      -D BUILD_SHARED_LIBS=ON \
```
See explanation elsewhere about static linking.